gradient/cli/common.py

```diff
@@ -171,3 +219,31 @@ def options_file(f):
         )
     ]
     return functools.reduce(lambda x, opt: opt(x), reversed(options), f)
+
```

Comment:
Please remember to have two empty lines between functions or classes definitions